UI/MapControlSample/MapControlSample/README.md

```diff
@@ -1,7 +1,11 @@
 # Uno Platform MapControl sample
 
-This is a small standalone sample illustrating how to use [MapControl](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Maps.MapControl) in an Uno Platform app. Currently MapControl is only supported on iOS and Android.
+This is a small standalone sample illustrating how to use [MapControl](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Maps.MapControl) in an Uno Platform app.
 
 You can find [detailed instructions on configuring map support in your app](https://platform.uno/docs/articles/controls/map-control-support.html) in the Uno Platform documentation.
 
-Note: to run this sample on Android, you'll need to [add your own API key](https://platform.uno/docs/articles/controls/map-control-support.html#get-api-key-for-the-map-component) first. You should add it [here](./MapControlSample.Droid/Properties/AssemblyInfo.cs).
\ No newline at end of file
+Note: to run this sample on Android, you'll need to [add your own API key](https://platform.uno/docs/articles/controls/map-control-support.html#get-api-key-for-the-map-component) first. You should add it [here](./MapControlSample.Droid/Properties/AssemblyInfo.cs).
```

Comment:
@nickrandolph This part seems to be outdated. However, I didn't find any information about the setup required or if it is needed at all. Do you care to update this part?

Reply:
I suspect the part about the API key is old too. Can you check with mapsui whether a key is required and update the link to point to their site if it it

Reply:
Ill remove the part, since I didn't find anything in the code either.